### PR TITLE
Dev: behave: Add functional test for cluster API

### PR DESCRIPTION
**Pull request by liangxin1300**
Test cases include:
- Start and stop resource
- crm resource move
- crm configure show
- pacemaker ACL related operations
- crm --force node fence \<target_node\>

### CODE REVIEW

**Comment by liangxin1300** [comment]
The last one `crm --force node fence <target_node>` will be executed on our HA local CI; Others will be executed on github actions

**Comment by liangxin1300** [comment]
Hi @fmherschel @zzhou1 
Could you please review this PR?
Thanks!

**Review by zzhou1 — COMMENTED** [review]
I prefer to add the reference tracking record in the commit log, something like the following,

REF: clusterAPI's pseudo version tag "v23.2022-11-15"

File: test/features/cluster_api.feature [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    Then    Resource "d" type "Dummy" is "Stopped"
+    And     Show cluster status on "hanode1"
+    When    Run "su hacluster -c 'crm resource start d'" on "hanode1"
+    Then    Expected return code is "0"
+    When    Wait "3" seconds
+    Then    Resource "d" type "Dummy" is "Started"
+    And     Show cluster status on "hanode1"
+
+  @clean
+  Scenario: Resource move by hacluster
```
[the remark this answers is not in the document]
Comment (liangxin1300) [reply test/features/cluster_api.feature:36]: Changed

**Comment by fmherschel** [comment]
> Hi @fmherschel @zzhou1 Could you please review this PR? Thanks!

@liangxin1300 Technically (on code base) I could not review this PR. I could assist, if you like me and Lars to review your concept. But this should be done via eMail, SUSE internally.

**Review by zzhou1 — APPROVED** [review]

**Pull request merged**